tests/index.js

```diff
@@ -75,22 +106,22 @@ describe('rule documentation files have the correct content', () => {
 
       // Ensure that unexpected notices are not present.
       unexpectedNotices.forEach((unexpectedNotice) => {
-        assert.ok(!documentContents.includes(MESSAGES[unexpectedNotice]), `does not include unexpected ${unexpectedNotice} notice`);
+        assert.ok(!arrayIncludes(documentContents, MESSAGES[unexpectedNotice]), `does not include unexpected ${unexpectedNotice} notice`);
```

Comment:
These are all [string includes](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/includes) which is available as of Node 4 so we should be able to use it.